azure-pipelines.yml

```diff
@@ -92,7 +92,7 @@ variables:
       value: 'azure-control-stg-rg'
     ${{ if eq( parameters['env'], 'ptlsbox') }}:
       value: 'azure-control-sbox-rg'
-    ${{ if notIn( parameters['env'], 'aat', 'perftest', 'ptlsbox', 'preview') }}:
+    ${{ if notIn( parameters['env'], 'aat', 'perftest', 'preview') }}:
```

Comment:
don't you need to remove line 93 and 94 too?

Reply:
:D literally commented same

Reply:
Removed now :)